components/TextFields/src/MDCTextInputControllerOutlined.m

```diff
@@ -60,13 +60,13 @@ - (void)setFloatingEnabled:(__unused BOOL)floatingEnabled {
   // Unused. Floating is always enabled.
 }
 
-- (CGPoint)floatingPlaceholderDestination {
-  CGPoint destination = [super floatingPlaceholderDestination];
+- (UIOffset)floatingPlaceholderOffset {
+  UIOffset destination = [super floatingPlaceholderOffset];
```

Comment:
Nit: Rename `destination` since it's now an offset?

Reply:
Done